frontend/javascripts/components/MenuView.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Link } from 'react-router'
+import { Link, Route } from 'react-router-dom'
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'Route' is defined but never used. (no-unused-vars)